docs/manifest.json

```
@@ -2444,7 +2444,7 @@
                         },
                         {
                           "title": "Add custom sign up and sign in pages",
-                          "href": "/docs/references/react-router/custom-signup-signin-pages"
+                          "href": "/docs/references/react-router/custom-sign-up-signin-pages"
```

https://clerk.com/docs/references/react-router/custom-signup-signin-pages

```suggestion
                          "href": "/docs/references/react-router/custom-signup-signin-pages"
```

---

also, we do we need to update this guide how we did the nextjs, expo, remix, and tanstack ones?

---

Yup, will get this updated!

---

@alexisintech here is a PR for React Router updates https://github.com/clerk/clerk-docs/pull/1821